README.md

```diff
@@ -32,6 +32,32 @@ export VENV=tie-breaker-venv
 ./run_job_mix.sh --help
 ```
 
+### To run distributed tiebreaker setup
+This assume every node in the cluster has the virtual environment setup complete. See Setup Environment above. 
+
+On **each node**, run the following
+```
+export USE_SUDO=1
+export VENV=~/tie-breaker-venv
+source ~/tie-breaker-venv/bin/activate
+```
+Designate one node to host the scheduler and kafka. This node can also run a TieBreaker controller instance. 
```

Comment:
This node can also run a TieBreaker controller instance. Could you please explain this line? Was it supposed to be scheduler instance?